backend/app/ingest.py

```
@@ -621,11 +652,105 @@ async def list_jobs():
+            status_code=500,
+            detail=f"Failed to delete document chunks from vector database: {e}"
+        )
+
+    # 4. Delete uploaded file from filesystem
+    file_path = Path(job.get("file_path", ""))
+    if file_path.exists():
+        try:
+            file_path.unlink()
+            result["file_deleted"] = True
+            logger.info(f"[{job_id}] Deleted file: {file_path}")
+        except Exception as e:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)